models/channel/src/presets/clothes-store.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { TChannelGraphql, TChannelRest } from '../types';
+import { restPreset, graphqlPreset, compatPreset } from './clothes-store';
+
+const restExpectation = (channel: TChannelRest) => {
+  expect(channel).toMatchObject({
+    key: 'food-store-key',
```

Comment:
also just cosmetics but maybe key should be `clothes-store-key`

Reply:
That's indeed true.

Updated here: e883da557370cb1ba1f63aadb99c2e31fce50120